addons/event/models/event.py

```diff
@@ -259,6 +266,16 @@ def mail_attendees(self, template_id, force_send=False, filter_func=lambda self:
         for attendee in self.registration_ids.filtered(filter_func):
             self.env['mail.template'].browse(template_id).send_mail(attendee.id, force_send=force_send)
 
+    @api.multi
+    def send_mail_all(self):
+        """ send email each attendees for change event Schedule"""
+        template = self.env.ref('event.event_change_notification')
+        status = ''
+        for event in self.filtered(lambda e: e.state == 'confirm' or e.state == 'cancel'):
+            event.schedule_change = False
+            status = 'open' if event.state == 'confirm' else event.state
+        self.mail_attendees(template.id, force_send=True, filter_func=lambda e: e.state == status)
```

Comment:
it's problem in status. status will be consider for last event. it seem wrong...